seed/challenges/01-responsive-web-design/basic-html-and-html5.json

```diff
@@ -846,7 +846,7 @@
         "Sometimes you want to add <code>a</code> elements to your website before you know where they will link.",
         "This is also handy when you're changing the behavior of a link using <code>jQuery</code>, which we'll learn about later.",
         "<hr>",
-        "The current value of the <code>href</code> attribute is a link that points to \"http://freecatphotoapp.com\". Replace the <code>href</code> attribute value with a <code>#</code>, also known as a hash symbol, to create a dead link."
+        "The current value of the <code>href</code> attribute is a link that points to \"http://freecatphotoapp.com\". Replace the <code>href</code> attribute value with a <code>#</code>, also known as a hash symbol, to create a dead link.For example: <code>href</code>=\"#\""
```

Comment:
Take your change to a new line as shown in the opening thread, also the formatting is not as expected:

> Replace the value of your `a` element's `href` attribute with a `#`, also known as a hash symbol, to turn it into a dead link.
> For example: `href="#"`

You can do this by adding moving the `code` block in your change to its correct location and taking the whole line to a new line by adding it an set of `""`, also remember to add a `,` in the previous line when you have separated the changes to a new line.